topics/auth0/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: identity, id-as-a-service, idaas
+created_by: Eugenio Pace and Matías Woloski
+display_name: Auth0
+github_url: https://github.com/Auth0
+logo: auth0.png
+related: single-sign-on, token-based-authentication, json-web-token, openid-connect, oauth, ws-federation, saml, identity-providers, sdk, enterprise-integration 
+released: 2013
+short_description: Auth0 is the new way to solve Identity. 
+topic: auth0
+url: https://auth0.com
+---
+Auth0 is a global leader in Identity-as-a-Service (IDaaS), provides thousands of enterprise customers with a Universal Identity Platform for their web, mobile, IoT, and internal applications. Its extensible platform seamlessly authenticates and secures more than 1.5B logins per month, making it loved by developers and trusted by global enterprises.
```

Comment:
See https://travis-ci.org/github/explore/builds/365937424?utm_source=github_status&utm_medium=notification

Reply:
@MikeMcQuaid Thanks, I saw it. To my understanding it doesn't like `0` character in the company name `Auth0`. I can rename it to `AuthZero` or `Authzero` or something else, but that will be another company, not Auth0.

Is there a trick to escape the `Auth0` to pass the validation? For example, extend the list of `["Perl", "Pi"]` with value `Auth`.

Reply:
hi logically it s free for your exemple

Reply:
@olegburov You could edit this line: https://github.com/github/explore/blob/1a3a268ac614432e7818c38be6ec38292558032b/test/topics_test.rb#L328